### PR TITLE
Update known-issues.md

### PR DESCRIPTION
**Pull request by andreighita**
This is tracked by the calendarView API team under reference: 1044260

### CODE REVIEW

**Comment by Lauragra** [comment]
@angelgolfer-ms, can you review this please?

File: concepts/known-issues.md [hunk 1]
```diff
@@ -169,6 +169,10 @@ Currently, the **onlineMeetingUrl** property of a Skype meeting [event](/graph/a
 
 - Navigation path `/applications/{id}` is not supported. Navigating through the global applications node to the application, even your own, is not allowed. Please use the `/app` navigation only.
 
+## Incorrect values for createdDateTime and lastModifiedDateTime
```
Comment (Lauragra) [inline concepts/known-issues.md]: This section needs to be moved to an H3 under the Calendar H2.

**Comment by Lauragra** [comment]
@angelgolfer-ms, FYI, do we need to add a Note to our docs about this? @andreighita, is there a timeline for this issue to be fixed?

**Comment by angelgolfer-ms** [comment]
@andreighita , let's follow up by internal email.

**Pull request closed without merging**